action-allowedlist/src/logging.psm1

```diff
@@ -0,0 +1,66 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+# This logging module writes messages intended for consumption by GitHub.
+# https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions
+
+$log = [System.Collections.ArrayList]::new()
+$errorOccurred = $false
+
+function Write-DebugLog {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory = $True)]
+        [string]
+        $Message
+    )
+
+    $log.Add("::debug::$Message") | Out-Null
+
+}
+
+function Write-InfoLog {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory = $True)]
+        [string]
+        $Message
+    )
+
+    $log.Add($Message) | Out-Null
+}
+
+function Write-WarnLog {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory = $True)]
+        [string]
+        $Message
+    )
+
+    $log.Add("::warning::$Message") | Out-Null
```

Comment:
In a future iteration, it might be interesting to track the line number from the YAML file that contains the error. It can be reported here in the warning message. Too much work for right now though.